### DISCUSSION TITLE
2.8.6

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#486] [Releases]
# Proudly releasing v2.8.6 to fix a single bug #485 🫡
<img width="375" alt="image" src="https://github.com/upscayl/upscayl/assets/25067102/741e7713-12d8-4f9c-b00b-0c76907798a5">

<hr /><em>This discussion was created from the release <a href='https://github.com/upscayl/upscayl/releases/tag/v2.8.6'>2.8.6</a>.</em>

**Comment by IkelAtomig** [comment]
Not in Flatpak yet.

**Reply by NayamAmarshe** [reply]
Oh yeah, the flatpak build failed. I need to fix that

**Comment by phamtq** [comment]
🫡 You do good work

**Comment by sebastionjenson** [comment]
Forced update breaks default (previously selected) model

⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"Sharpen Image","value":"models-DF2K"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🔄 Resetting image paths
🔀 Setting model to models-DF2K
🐞 BACKEND REPORTED:  💾 Updating Save Output Folder:  false
🐞 BACKEND REPORTED:  📐 Updating Compression:  0
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
🔄 Resetting image paths
⤵️ Dropped file:  {"type":"image/png","filePath":"C:\\Users\\seb\\Desktop\\asdf.png","extension":"png"}
🖼 Setting image path:  C:\Users\seb\Desktop\asdf.png
🗂 Setting output path:  C:\Users\seb\Desktop
🖼 imagePath:  C:\Users\seb\Desktop\asdf.png
🔤 Extension:  png
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
🐞 BACKEND REPORTED:  ✅ Upscayl Variables:  [object Object]
🐞 BACKEND REPORTED:  📢 Upscayl Command:  -i,C:\Users\seb\Desktop\asdf.png,-o,C:\Users\seb\Desktop\asdf_upscayl_2x_models-DF2K.png,-s,4,-m,C:\Program Files\Upscayl\resources\models,-n,models-DF2K,,,-f,png
🐞 BACKEND REPORTED:  👶 Updating Child Processes:  [object Object]
🐞 BACKEND REPORTED:  🛑 Updating Stopped:  false
🐞 BACKEND REPORTED:  image upscayl:  [0 NVIDIA GeForce RTX 3060 Ti]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 3060 Ti]  bugsbn1=0  bugbilz=160  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 3060 Ti]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 3060 Ti]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 NVIDIA GeForce RTX 3060 Ti]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 3060 Ti]  bugsbn1=0  bugbilz=160  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 3060 Ti]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 3060 Ti]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  _wfopen C:\Program Files\Upscayl\resources\bin\C:\Program Files\Upscayl\resources\models/models-DF2K.param failed

🔄 Resetting image paths
🚧 UPSCAYL_PROGRESS:  _wfopen C:\Program Files\Upscayl\resources\bin\C:\Program Files\Upscayl\resources\models/models-DF2K.param failed

🔄 Resetting image paths
🐞 BACKEND REPORTED:  ❌ INVALID GPU OR FAILED
🐞 BACKEND REPORTED:  📁 Updating Folder Path:  C:\Users\seb\Desktop
🗂 Setting Output Path:  C:\Users\seb\Desktop
🧠 Remembering Output Folder:  C:\Users\seb\Desktop
🐞 BACKEND REPORTED:  📁 Selected Folder Path:  C:\Users\seb\Desktop
🔄 Resetting Upscaled Image Path
🚫 No valid image selected
🔄 Resetting image paths
⤵️ Dropped file:  {"type":"image/png","filePath":"C:\\Users\\seb\\Desktop\\asdf.png","extension":"png"}
🖼 Setting image path:  C:\Users\seb\Desktop\asdf.png
🗂 Setting output path:  C:\Users\seb\Desktop
🖼 imagePath:  C:\Users\seb\Desktop\asdf.png
🔤 Extension:  png
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
🐞 BACKEND REPORTED:  ✅ Upscayl Variables:  [object Object]
🐞 BACKEND REPORTED:  📢 Upscayl Command:  -i,C:\Users\seb\Desktop\asdf.png,-o,C:\Users\seb\Desktop\asdf_upscayl_2x_models-DF2K.png,-s,4,-m,C:\Program Files\Upscayl\resources\models,-n,models-DF2K,,,-f,png
🐞 BACKEND REPORTED:  👶 Updating Child Processes:  [object Object],[object Object]
🐞 BACKEND REPORTED:  🛑 Updating Stopped:  false
🐞 BACKEND REPORTED:  image upscayl:  [0 NVIDIA GeForce RTX 3060 Ti]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 3060 Ti]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 3060 Ti]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 3060 Ti]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 NVIDIA GeForce RTX 3060 Ti]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 3060 Ti]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 3060 Ti]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 3060 Ti]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  _wfopen C:\Program Files\Upscayl\resources\bin\C:\Program Files\Upscayl\resources\models/models-DF2K.param failed

🔄 Resetting image paths
🚧 UPSCAYL_PROGRESS:  _wfopen C:\Program Files\Upscayl\resources\bin\C:\Program Files\Upscayl\resources\models/models-DF2K.param failed

🔄 Resetting image paths
🐞 BACKEND REPORTED:  ❌ INVALID GPU OR FAILED
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"Sharpen Image","value":"models-DF2K"}
⚙️ Setting gpuId to empty string
⚙️ Getting rememberOutputFolder from localStorage:  false
⚙️ Getting compression from localStorage 0
🐞 BACKEND REPORTED:  🚫 Select Custom Models Folder Operation Cancelled
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"Sharpen Image","value":"models-DF2K"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🔀 Setting model to models-DF2K
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"Sharpen Image","value":"models-DF2K"}
⚙️ Setting gpuId to empty string
⚙️ Getting rememberOutputFolder from localStorage:  false
⚙️ Getting compression from localStorage 0

**Reply by NayamAmarshe** [reply]
Just change the model.

**Comment by jschwalbe** [comment]
Anyone try Upscayl'ing the photo in the post? Yikes! haha
